biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -2321,14 +2342,20 @@ subroutine fuse_patches( csite, bc_in )
                             endif ! sum(biomass(:,:) .gt. force_patchfuse_min_biomass 
                          endif ! maxage
 
+
+                         ! Do not fuse patches that have different PFT labels in nocomp mode
```

Comment:
should we have this fusion exclusion prior to the the block above?  It seems there is no reason to calculate and compare age profiles if the patches are in nocomp and have different pfts

https://github.com/NGEET/fates/pull/703/files?diff=unified&w=1#diff-28c679efbe804905b8b4fac3ccdbb04f6bb3bcd2794fe1a4556d5541e008a750R2289-R2343

Reply:
also, this whole routine would benefit from using named "if" "do" and "while"s.  Its starting to get hard to determine where things are opening and closing.

Reply:
examples:
open: https://github.com/NGEET/fates/blob/master/parteh/PRTAllometricCNPMod.F90#L1399
close: https://github.com/NGEET/fates/blob/master/parteh/PRTAllometricCNPMod.F90#L1534